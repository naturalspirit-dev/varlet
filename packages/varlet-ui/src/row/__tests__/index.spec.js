import example from '../example'
import { render } from '@testing-library/vue'

test('test row example', async () => {
  const wrapper = render(example)
  console.log(wrapper)
})
