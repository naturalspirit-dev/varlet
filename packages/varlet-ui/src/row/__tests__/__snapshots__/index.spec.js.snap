// Vitest Snapshot v1

exports[`test row and col example 1`] = `
"<div class=\\"app-type\\" data-v-5c0415cd=\\"\\">栅格系统</div>
<div class=\\"var-row var--box\\" style=\\"justify-content: flex-start; align-items: flex-start;\\" data-v-5c0415cd=\\"\\">
  <div class=\\"var-col var--box var-col--span-8\\" style=\\"padding-left: 0px; padding-right: 0px;\\" data-v-5c0415cd=\\"\\">span: 8</div>
  <div class=\\"var-col var--box var-col--span-8\\" style=\\"padding-left: 0px; padding-right: 0px;\\" data-v-5c0415cd=\\"\\">span: 8</div>
  <div class=\\"var-col var--box var-col--span-8\\" style=\\"padding-left: 0px; padding-right: 0px;\\" data-v-5c0415cd=\\"\\">span: 8</div>
  <div class=\\"var-col var--box var-col--span-8\\" style=\\"padding-left: 0px; padding-right: 0px;\\" data-v-5c0415cd=\\"\\">span: 8</div>
  <div class=\\"var-col var--box var-col--span-8\\" style=\\"padding-left: 0px; padding-right: 0px;\\" data-v-5c0415cd=\\"\\">span: 8</div>
  <div class=\\"var-col var--box var-col--span-8\\" style=\\"padding-left: 0px; padding-right: 0px;\\" data-v-5c0415cd=\\"\\">span: 8</div>
</div>
<div class=\\"app-type\\" data-v-5c0415cd=\\"\\">偏移</div>
<div class=\\"var-row var--box\\" style=\\"justify-content: flex-start; align-items: flex-start;\\" data-v-5c0415cd=\\"\\">
  <div class=\\"var-col var--box var-col--span-16 var-col--offset-8\\" style=\\"padding-left: 0px; padding-right: 0px;\\" data-v-5c0415cd=\\"\\">offset: 8 span: 16</div>
  <div class=\\"var-col var--box var-col--span-8\\" style=\\"padding-left: 0px; padding-right: 0px;\\" data-v-5c0415cd=\\"\\">span: 8</div>
  <div class=\\"var-col var--box var-col--span-8\\" style=\\"padding-left: 0px; padding-right: 0px;\\" data-v-5c0415cd=\\"\\">span: 8</div>
  <div class=\\"var-col var--box var-col--span-8\\" style=\\"padding-left: 0px; padding-right: 0px;\\" data-v-5c0415cd=\\"\\">span: 8</div>
</div>
<div class=\\"app-type\\" data-v-5c0415cd=\\"\\">对齐</div>
<div class=\\"var-row var--box\\" style=\\"justify-content: flex-start; align-items: flex-start;\\" data-v-5c0415cd=\\"\\">
  <div class=\\"var-col var--box var-col--span-8\\" style=\\"padding-left: 0px; padding-right: 0px;\\" data-v-5c0415cd=\\"\\">span: 8</div>
  <div class=\\"var-col var--box var-col--span-8\\" style=\\"padding-left: 0px; padding-right: 0px;\\" data-v-5c0415cd=\\"\\">span: 8</div>
</div>
<div class=\\"var-row var--box\\" style=\\"justify-content: center; align-items: flex-start;\\" data-v-5c0415cd=\\"\\">
  <div class=\\"var-col var--box var-col--span-8\\" style=\\"padding-left: 0px; padding-right: 0px;\\" data-v-5c0415cd=\\"\\">span: 8</div>
  <div class=\\"var-col var--box var-col--span-8\\" style=\\"padding-left: 0px; padding-right: 0px;\\" data-v-5c0415cd=\\"\\">span: 8</div>
</div>
<div class=\\"var-row var--box\\" style=\\"justify-content: flex-end; align-items: flex-start;\\" data-v-5c0415cd=\\"\\">
  <div class=\\"var-col var--box var-col--span-8\\" style=\\"padding-left: 0px; padding-right: 0px;\\" data-v-5c0415cd=\\"\\">span: 8</div>
  <div class=\\"var-col var--box var-col--span-8\\" style=\\"padding-left: 0px; padding-right: 0px;\\" data-v-5c0415cd=\\"\\">span: 8</div>
</div>
<div class=\\"var-row var--box\\" style=\\"justify-content: space-around; align-items: flex-start;\\" data-v-5c0415cd=\\"\\">
  <div class=\\"var-col var--box var-col--span-8\\" style=\\"padding-left: 0px; padding-right: 0px;\\" data-v-5c0415cd=\\"\\">span: 8</div>
  <div class=\\"var-col var--box var-col--span-8\\" style=\\"padding-left: 0px; padding-right: 0px;\\" data-v-5c0415cd=\\"\\">span: 8</div>
</div>
<div class=\\"var-row var--box\\" style=\\"justify-content: space-between; align-items: flex-start;\\" data-v-5c0415cd=\\"\\">
  <div class=\\"var-col var--box var-col--span-8\\" style=\\"padding-left: 0px; padding-right: 0px;\\" data-v-5c0415cd=\\"\\">span: 8</div>
  <div class=\\"var-col var--box var-col--span-8\\" style=\\"padding-left: 0px; padding-right: 0px;\\" data-v-5c0415cd=\\"\\">span: 8</div>
</div>
<div class=\\"app-type\\" data-v-5c0415cd=\\"\\">列间距</div>
<div class=\\"var-row var--box\\" style=\\"justify-content: flex-start; align-items: flex-start; margin: 0px -5px;\\" data-v-5c0415cd=\\"\\">
  <div class=\\"var-col var--box var-col--span-8\\" style=\\"padding-left: 0px; padding-right: 0px;\\" data-v-5c0415cd=\\"\\">span: 8</div>
  <div class=\\"var-col var--box var-col--span-8\\" style=\\"padding-left: 0px; padding-right: 0px;\\" data-v-5c0415cd=\\"\\">span: 8</div>
  <div class=\\"var-col var--box var-col--span-8\\" style=\\"padding-left: 0px; padding-right: 0px;\\" data-v-5c0415cd=\\"\\">span: 8</div>
  <div class=\\"var-col var--box var-col--span-8\\" style=\\"padding-left: 0px; padding-right: 0px;\\" data-v-5c0415cd=\\"\\">span: 8</div>
  <div class=\\"var-col var--box var-col--span-8\\" style=\\"padding-left: 0px; padding-right: 0px;\\" data-v-5c0415cd=\\"\\">span: 8</div>
  <div class=\\"var-col var--box var-col--span-8\\" style=\\"padding-left: 0px; padding-right: 0px;\\" data-v-5c0415cd=\\"\\">span: 8</div>
</div>"
`;
